gremlin-test/features/map/Edge.feature

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassMap @StepE
+Feature: Step - E()
```

Comment:
Few questions on the cucumber suite:
1. Should we add additional E() tests? It seems to be a big function addition with 1 test to cover
2. Should we move the other E() tests from the `Vertex.feature` file over to here?
3. Mid-traversal V() appears to be a large part of the `Graph.feature` tests, should mid-traversal E() tests be added there too?

Reply:
`Vertex.feature` - Step - V(), E(), out(), in(), both(), inE(), outE(), bothE()
`Graph.feature` - mid traversal V(), separated into other file so that you can run it separately
`Edge.feature` - same for mid traversal E()